pensieve/cli.py

```diff
@@ -139,3 +142,12 @@ def rerun(project_id, dataset_id, experiment_slug, dry_run, config_file):
 
     for date in inclusive_date_range(experiment.start_date, end_date):
         Analysis(project_id, dataset_id, config).run(date, dry_run=dry_run)
+
+
+@cli.command()
+@project_id_option
+@dataset_id_option
+@bucket_option
+def export_json(project_id, dataset_id, bucket):
```

Comment:
We might want to rename this command to something else, though `export_statistics_to_json` also doesn't sound to great.

Reply:
`export_statistics` seems good.

Reply:
> We might want to rename this command to something else, though `export_statistics_to_json` also doesn't sound to great.

I'm not super picky so it's up to you. But in my opinion, more specific naming is often more clear. So despite `export_statistics_to_json` being wordy, it's clear exactly what it does.